bugsnag-android-core/src/main/java/com/bugsnag/android/internal/Journal.kt

```diff
@@ -0,0 +1,151 @@
+package com.bugsnag.android.internal
+
+import com.bugsnag.android.Logger
+import com.dslplatform.json.DslJson
+import java.io.IOException
+import java.io.OutputStream
+import java.util.*
```

Comment:
There seem to be a couple of style tweaks needed here that are currently failing CI